README.md

```diff
@@ -627,7 +627,20 @@ And then depending on if you are using solo mining or self-select mining you wil
 - For the Tari Merge Mining Proxy, under section **`merge_mining_proxy.weatherwax`**
   ```
   [merge_mining_proxy.weatherwax]
-  monerod_url = "http://monero-stagenet.exan.tech:38081"
+  monerod_url = [ # stagenet
```

Comment:
There is a lot of repetition of these values here in the README. In fact I see there are quite a lot of config option in this document that I think shouldn't be here for cleanliness sake. I propose you put these options into the sample config file in the correct places, with some comments on what they are and then in the Readme just discuss them if they need to be discussed.